extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -57,7 +57,7 @@ export class AmpIframe extends AMP.BaseElement {
     user().assert(
         url.protocol == 'https:' ||
         url.protocol == 'data:' ||
-        url.origin.indexOf('http://iframe.localhost:') == 0,
+        /http:\/\/(\w*)\.{0,1}localhost(:|\/)(.*)/.test(url.origin),
```

Comment:
We can actually change all of this to use [`isSecureHost`](https://github.com/ampproject/amphtml/blob/master/src/url.js#L190-L202):

```js
user().assert(isSecureHost(url) || url.protocol == 'data:', ...)